### PR TITLE
add : moreWork(); to (a)synchronous example

### PR DESCRIPTION
**Pull request by hanarotg**
moreWork(); missed in expanded (a)synchronous example code

### CODE REVIEW

File: locale/ko/docs/guides/blocking-vs-non-blocking.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ moreWork(); // will run before console.log
 const fs = require('fs');
 const data = fs.readFileSync('/file.md'); // 파일을 읽을 때까지 여기서 블로킹됩니다.
 console.log(data);
-// moreWork();는 console.log 이후 실행될 것입니다.
+moreWork(); // moreWork();는 console.log 이후 실행될 것입니다.
```
Comment (nschonni) [inline locale/ko/docs/guides/blocking-vs-non-blocking.md:144]:
```suggestion
moreWork(); // 는 console.log 이후 실행될 것입니다.
```

File: locale/ko/docs/guides/blocking-vs-non-blocking.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ fs.readFile('/file.md', (err, data) => {
   if (err) throw err;
   console.log(data);
 });
-// moreWork();는 console.log 이전에 실행될 것입니다.
+moreWork(); // moreWork();는 console.log 이전에 실행될 것입니다.
```
Comment (nschonni) [inline locale/ko/docs/guides/blocking-vs-non-blocking.md]:
```suggestion
moreWork(); // 는 console.log 이전에 실행될 것입니다.
```

**Review by nschonni — COMMENTED** [review]
Ping @nodejs/nodejs-ko

File: locale/ko/docs/guides/blocking-vs-non-blocking.md (hunk 1 above)
Comment (yous) [reply locale/ko/docs/guides/blocking-vs-non-blocking.md:144]: I would prefer this:
```suggestion
moreWork(); // 이는 console.log 이후 실행될 것입니다.
```
How do you think? @nodejs/nodejs-ko

File: locale/ko/docs/guides/blocking-vs-non-blocking.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ fs.readFile('/file.md', (err, data) => {
   if (err) throw err;
   console.log(data);
 });
-// moreWork();는 console.log 이전에 실행될 것입니다.
+moreWork(); // 는 console.log 이전에 실행될 것입니다.
```
Comment (yous) [inline locale/ko/docs/guides/blocking-vs-non-blocking.md:155]:
```suggestion
moreWork(); // 이는 console.log 이전에 실행될 것입니다.
```

**Comment by ovflowd** [comment]
Closing as a stale PR.

**Pull request closed without merging**